Examples/Algorithms/Geant4/src/MMPrimaryGeneratorAction.cpp

```diff
@@ -6,18 +6,17 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-#include "ACTFW/Plugins/Geant4/MMPrimaryGeneratorAction.hpp"
+#include "MMPrimaryGeneratorAction.hpp"
```

Comment:
Why relative includes here? Same for the other cases after this?

Reply:
Because they are internal files now and not part of the public interface anymore.

Reply:
Does not apply anymore since the relevant changes have been removed from this PR.